ZuluContent/Engines/Harvest/Core/HarvestSystem.cs

```diff
@@ -437,6 +439,7 @@ public virtual void StartHarvesting(Mobile from, Item tool, object toHarvest)
             }
 
             new HarvestTimer(from, tool, this, def, toHarvest, toLock).Start();
+            from.NextSkillTime = Core.TickCount + ((int) def.EffectDelay.TotalMilliseconds * def.EffectCounts[0]) + 7000;
```

Comment:
Can't use the `HarvestTimer.Delay` here instead of calculating again? It might be off if there's more than one EffectCounts entry as well.

Also what is the magic `7000` delay? Should that have a name/and or be configured via JSON?

Reply:
Yeah good catch, I intended to refactor this but must've forgotten about it... the 7000 delay is for the 6000 ms delay between loops + 1000 ms delay when it waits before rolling for the bonus harvest effects like finding zulu ore, deadwood, magic fish etc.

Will clean this up to be configured better